sdk/test/trace/span_data_test.cc

```diff
@@ -69,7 +69,8 @@ TEST(SpanData, Set)
   data.SetStartTime(now);
   data.SetDuration(std::chrono::nanoseconds(1000000));
   data.SetAttribute("attr1", static_cast<int64_t>(314159));
-  data.AddEvent("event1", now);
+  data.AddEvent("event1", now,
+                opentelemetry::common::KeyValueIterableView<std::map<std::string, int32_t>>({}));
```

Comment:
```suggestion
  data.AddEvent("event1", now);
```

Reverting to original commit